libraries/AP_Logger/LogStructure.h

```diff
@@ -1571,7 +1620,13 @@ LOG_STRUCTURE_FROM_VISUALODOM \
     { LOG_PSC_MSG, sizeof(log_PSC), \
       "PSC", "Qffffffffffff", "TimeUS,TPX,TPY,PX,PY,TVX,TVY,VX,VY,TAX,TAY,AX,AY", "smmmmnnnnoooo", "F000000000000" }, \
     { LOG_PSCZ_MSG, sizeof(log_PSCZ), \
-      "PSCZ", "Qfffffffff", "TimeUS,TPZ,PZ,DVZ,TVZ,VZ,DAZ,TAZ,AZ,ThO", "smmnnnooo%", "F000000002" }
+      "PSCZ", "Qfffffffff", "TimeUS,TPZ,PZ,DVZ,TVZ,VZ,DAZ,TAZ,AZ,ThO", "smmnnnooo%", "F000000002" }, \
+    { LOG_AIS_RAW_MSG, sizeof(log_AIS_raw), \
+      "AISR",  "QBBBZ", "TimeUS,num,total,ID,payload", "s----", "F0000" }, \
+    { LOG_AIS_MSG1, sizeof(log_AIS_msg1), \
+      "AIS1",  "QBBIBbHbLLHHBBbI", "US,typ,rep,mmsi,nav,rot,sog,pos,lon,lat,cog,hed,sec,man,raim,rad", "s---------------", "F000000000000000" }, \
+    { LOG_AIS_MSG5, sizeof(log_AIS_msg5), \
+      "AIS5",  "QBIBINZBHHBBBBZB", "US,rep,mmsi,ver,imo,cal,nam,typ,bow,stn,prt,str,fix,dght,dst,dte", "s-------mmmm-m--", "F------------A--" }
```

Comment:
These should be put into the AIS directory in the same way as we are for many other libraries now - see AP_NavEKF2 for example.

Additionally, documentation should be added for these messages.

Reply:
moved and added docs